roles/cgit/templates/docker-compose.yml.j2

```diff
@@ -20,22 +20,14 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.docker.network={{ traefik_external_network_name }}"
-      - "traefik.http.services.cgit.loadbalancer.server.port=80"
       - "traefik.http.middlewares.cgit-redirect-to-https.redirectscheme.scheme=https"
       - "traefik.http.routers.cgit.entrypoints=http"
-{% if cgit_traefik_host|length > 0 %}
-      - "traefik.http.routers.cgit.rule=Host(`{{ cgit_traefik_host }}`) && PathPrefix(`/{{ cgit_traefik_path_prefix }}/`)"
-{% else %}
-      - "traefik.http.routers.cgit.rule=PathPrefix(`/{{ cgit_traefik_path_prefix }}/`)"
-{% endif %}
+      - "traefik.http.routers.cgit.rule=Host(`{{ cgit_traefik_host }}`)"
```

Comment:
I wonder whether we could simply reuse the cgit_host var here and similarly for all other services. Is there a reason why we would need another variable?